pantab/tests/conftest.py

```diff
@@ -107,7 +107,7 @@ def df():
             "int32": np.int32,
             "Int32": "Int32",
             "int64": np.int64,
-            "Int32": "Int32",
+            "Int64": "Int64",
```

Comment:
By the way noticed this on other work. Thanks for fixing up here - could also in the future do as separate PRs